app/web/src/features/auth/email/ChangeEmail.tsx

```diff
@@ -11,16 +11,24 @@ import { useMutation } from "react-query";
 import { service } from "service";
 import { lowercaseAndTrimField } from "utils/validation";
 
+import makeStyles from "../../../utils/makeStyles";
 import useChangeDetailsFormStyles from "../useChangeDetailsFormStyles";
 
+const useStyles = makeStyles((theme) => ({
+  root: {
+    margin: theme.spacing(4, 0),
```

Comment:
Avoid root level margins because it affects things outside this component and is more difficult to maintain/reason about. That probably means adapting these components to accept `className` prop and using that in the parent. Slightly more work now, but much easier in the long run!

Reply:
Changed it in last push